src/markdown/getting-started/install.md

```diff
@@ -6,42 +6,73 @@ position: 1
 
 First of all, you have to install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) (standalone version is prefered) on your computer.
```

Comment:
I myself think thoses external link must be open in a new browser tab.

Reply:
I've changed them in order to let the user choose since it's the best practice